src/consensus/blockchain.py

```diff
@@ -445,14 +439,14 @@ def get_recent_reward_challenges(self) -> List[Tuple[bytes32, uint128]]:
                 for rc in reversed(curr.finished_reward_slot_hashes):
                     recent_rc.append((rc, sub_slot_total_iters))
                     sub_slot_total_iters = uint128(sub_slot_total_iters - curr.sub_slot_iters)
-            curr = self.sub_blocks.get(curr.prev_hash, None)
+            curr = self.__sub_blocks.get(curr.prev_hash, None)
```

Comment:
Why do we use .get in some places and try_sub_block in others? We should be consistent